Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/FindPeakAutomatic.py

```diff
@@ -0,0 +1,395 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+from mantid.simpleapi import (ConvertToPointData, CreateWorkspace, DeleteWorkspace,
+                              CreateEmptyTableWorkspace, FitGaussianPeaks)
+from mantid.api import (DataProcessorAlgorithm, AlgorithmFactory, WorkspaceProperty, Progress, ITableWorkspaceProperty)
+from mantid.kernel import (Direction, FloatBoundedValidator, IntBoundedValidator)
+from mantid import mtd, logger
+
+import numpy as np
+import scipy.signal
+
+
+class FindPeakAutomatic(DataProcessorAlgorithm):
```

Comment:
To keep more inline with the names of other algorithms, as I think this finds multiple peaks, can we call this `FindPeaksAutomatic`?